gcp/rakefiles/vars.rb

```diff
@@ -16,20 +16,31 @@ def self.set_vars(env, project_type)
       end
     end
 
-    if ENV["TF_VAR_project_id"].nil?
+
+    if ENV["TF_VAR_project_name"].nil?
       if ["dev"].include?(env)
-        ENV["TF_VAR_project_id"] = "gpii-#{project_type}-#{env}-#{ENV["USER"]}"
+        ENV["TF_VAR_project_name"] = "gpii-#{project_type}-#{env}-#{ENV["USER"]}"
       elsif ["stg", "prd"].include?(env)
-        ENV["TF_VAR_project_id"] = "gpii-#{project_type}-#{env}"
+        ENV["TF_VAR_project_name"] = "gpii-#{project_type}-#{env}"
       else
-        puts "  ERROR: TF_VAR_project_id must be set!"
+        puts "  ERROR: TF_VAR_project_name must be set!"
         puts "  Usually, this value will be calculated for you, but you are"
         puts "  using an env I don't recognize: #{env}."
         puts "  Either pick a different env or, if you know what you're doing,"
-        puts "  you may override TF_VAR_project_id directly:"
-        puts "    export TF_VAR_project_id=<env>-<your name>"
+        puts "  you may override TF_VAR_project_name directly:"
+        puts "    export TF_VAR_project_name=<env>-<your name>"
         puts "  Do one of those and try again."
-        raise ArgumentError, "TF_VAR_project_id must be set"
+        raise ArgumentError, "TF_VAR_project_name must be set"
```

Comment:
You will need to modify the tests for `set_vars()` to account for this behavior change (and others).

Reply:
Will do, thanks, didn't look at the tests yet